node/test/DailyHabit.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { DailyHabit } from "../src/domain/entity/DailyHabit"
+
+describe('daily Habit', () => {
+  let newDailyHabit!: DailyHabit
+  beforeEach(() => {
+    newDailyHabit = new DailyHabit(new Date())
+  })
+
+  it('should have day', () => {
```

Comment:
Hey this spec sounds  like testing the Date class. 
I think we should test if date is an instance of Date class. And do not test `getDate()` and others

Also, I would prefer to name the `it` as should have a date